ci-operator/step-registry/insights-live/insights-live-commands.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+# install the insights secret to the cluster to report support data from the support-operator
+oc create -f "${CLUSTER_PROFILE_DIR}/insights-live.yaml"
```

Comment:
I think I saw a PR to put this into a dedicated secret earlier, so this should not read `CLUSTER_PROFILE_DIR` anymore but rather something exposed via `credentials`?

Reply:
Nice catch. I'll take a look.

Reply:
`${CLUSTER_PROFILE_DIR}` should be correct right? regarding: https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables

Reply:
But since the user has to add a `mount_path`, we need to make sure that the path with the step will be the same. Or do we have any default `mount_path` ?

Reply:
Setting mount path is a part of `credentials` config: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-custom-credentials